composer.json

```diff
@@ -26,21 +26,21 @@
         "php": ">=5.5.0",
         "helhum/typo3-console-plugin": "^1.5.0",
 
-        "typo3/cms-backend": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-core": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-extbase": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-extensionmanager": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-fluid": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-install": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-saltedpasswords": "^7.6 || >=8.3.0 <8.7",
-        "typo3/cms-scheduler": "^7.6 || >=8.3.0 <8.7",
+        "typo3/cms-backend": "^7.6 || ^8.7",
+        "typo3/cms-core": "^7.6 || ^8.7",
+        "typo3/cms-extbase": "^7.6 || ^8.7",
+        "typo3/cms-extensionmanager": "^7.6 || ^8.7",
+        "typo3/cms-fluid": "^7.6 || ^8.7",
+        "typo3/cms-install": "^7.6 || ^8.7",
+        "typo3/cms-saltedpasswords": "^7.6 || ^8.7",
+        "typo3/cms-scheduler": "^7.6 || ^8.7",
```

Comment:
ok, you can't have minimum requirements set to 8.7 here, but leave the travis tests build on lower TYPO3 versions